README.md

```diff
@@ -0,0 +1,9 @@
+## JabbR - v2
+JabbR is a chat application built with ASP.NET using SignalR.
+
+## Prerequisites
+To be able to contribute to the project, you need to following new bits:
+* Visual Studio 2014 - Link ?
```

Comment:
That should be VS2015 right? Link: http://www.visualstudio.com/en-us/downloads/visual-studio-2015-downloads-vs.aspx